.github/workflows/cd.yml

```diff
@@ -0,0 +1,24 @@
+name: CD
+
+on:
+  release:
+    types: [published]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
```

Comment:
Hey there :)
We want to avoid using public runners. Let's use prima's self hosted runners instead
https://www.notion.so/helloprima/Check-Github-Actions-runner-memory-cpu-usage-3110242851294ca5a7e5b6ed44ad5ea5?pvs=4